src/amo/pages/AddonInfo/index.js

```diff
@@ -123,7 +123,21 @@ export class AddonInfoBase extends React.Component<InternalProps> {
         infoType === ADDON_INFO_TYPE_EULA
           ? addonInfo.eula
           : addonInfo.privacyPolicy;
-      infoHtml = sanitizeHTML(nl2br(infoContent), ['br']);
+      infoHtml = sanitizeHTML(nl2br(infoContent), [
```

Comment:
We actually have a function that does this already (sorry for not noticing before), it's called [`sanitizeUserHTML`](https://github.com/mozilla/addons-frontend/blob/2018.11.22/src/core/utils/index.js#L120) and you can see an example of its use [here](https://github.com/mozilla/addons-frontend/blob/2018.11.22/src/amo/pages/Addon/index.js#L337).